frontend/src/utility.ts

```diff
@@ -44,3 +44,23 @@ export const formatTimestamp = (value: string, item: any) => {
   const timestamp = _.get(item, value, false);
   return timestamp ? moment(timestamp).format("D.MM.YYYY") : "";
 };
+
+const taskIdPattern = "^\\w+-\\d+-\\d+";
+
+const taskId = new RegExp(taskIdPattern);
+
+const flacFileFormat = new RegExp(`${taskIdPattern}.flac`);
+
+export const validateFlacFile = ({ name: fileName, type }: any) => {
+  const isValidFormat = fileName.match(flacFileFormat);
+  if (isValidFormat && type === "audio/flac") return isValidFormat;
+
+  throw new Error(
+      type === "audio/flac" ? "The file name does not start with task ID" : "File type must be flac"
+  );
+};
```

Comment:
This function looks cumbersome. Please simplify. It's better to throw an error twice than combining into one conditional statement. Just for the sake of readability.

Reply:
Still I don't like it. What I'd do is to test one condition and throw error if not met, then test another condition and throw error. And in the end I'd return `true`. Would be more readable. Would you agree?